misc/toml-parser/src/main/java/io/ballerina/toml/semantic/ast/TomlTransformer.java

```diff
@@ -457,4 +489,20 @@ public TomlNode transform(BoolLiteralNode boolLiteralNode) {
     public TomlNode transform(IdentifierLiteralNode identifierLiteralNode) {
         return new TomlUnquotedKeyNode(identifierLiteralNode.value().text(), getPosition(identifierLiteralNode));
     }
+
+    private TomlNodeLocation getLocationOfKeyEntryList(List<TomlKeyEntryNode> keys) {
+        if (keys.size() == 0) {
+            return null;
+        }
+        int startOffset = keys.get(0).location().textRange().startOffset();
```

Comment:
`keys.get(0).location()` is repeatedly used within the method. Shall we extract it to a variable?